dotnet/samples/Concepts/Prompty/PromptyFunction.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel;
+
+namespace Prompty;
```

Comment:
Is this the right namespace?

Reply:
Yes, this is the convention for all of the Concept sample tests